new-pipeline/dist.html

```diff
@@ -59,6 +59,8 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                 </h2>
                 <hr></hr>
                 <figure class="col-md-12">
+                    <div id="use-counter-link" style="text-align: center">
```

Comment:
Can we follow the style of dist-caption-link below?
It can be on a new line, but follow the font style and add an icon (maybe simply also `fa-info-circle`?).